.changes/menu-refactor.md

```diff
@@ -0,0 +1,11 @@
+---
+"tao": minor
+---
+
+**Breaking change**: New menu/tray API.
+
+System tray now expose `set_icon()` to update the tray icon after initialization. The `system_tray::SystemTrayBuilder` has been moved to the root of the package as a module and available on Windows, Linux and macOS, only when the `tray` feature is enabled. Windows expose a `remove()` function available with `SystemTrayExtWindows`.
```

Comment:
we should probably mention the `remove()` function somewhere in the docs and warn users that if they don't call it on windows, the icon won't be removed automatically.

Reply:
let me know if it make sense;
https://github.com/tauri-apps/tao/pull/77/commits/49fce6a87927b9bf40ec94a6b293b4cf7aeaf246#diff-de45fd7bc91c961341f8b97439d08aaaefe616342ef4fa5ab235fda4b938afd2R50

Reply:
perfect!